wiki/Legal/Terms/en.md

```diff
@@ -80,6 +80,20 @@ osu! does not endorse any User Submission or any opinion, recommendation, or adv
 osu! does not permit copyright infringing activities and infringement of intellectual property rights on its Website, and osu! will remove all Content and User Submissions if properly notified that such Content or User Submission infringes on another's intellectual property rights.
 osu! reserves the right to remove Content and User Submissions without prior notice.
 
+## YOUR RESPONSIBILITIES IN VOLUNTEER POSITIONS
+
+osu! contains a number of volunteer teams at any given point in time that are drawn directly from the community. These teams are often invested with extra permissions and access to functions that ordinary users do not have in order to facilitate the community-driven aspect of the game. These officially supported teams are as follows:
+
+- The Beatmap Nominator Group
+- The Tournament Staff
```

Comment:
This seems very broad to me. What do you consider tournament "staff"? Map selectors? Referees? Statisticians? Testers? Commentators?

Reply:
purple people on osu! internal is what i mean, we've called them "tournament staff", so that encompasses all of those roles

Reply:
agree that this needs a more precise wording i think, especially considering how this term as it stands seems to possibly apply to staffs of all tournaments while in theory it only refers to *The* Tournament Staff

also for better clarity linking to each group's usergroup would be really nice as well i think, something like this perhaps :

- [The Beatmap Nominators](https://osu.ppy.sh/groups/28) and [The Probationary Beatmap Nominators](https://osu.ppy.sh/groups/32)
- The Tournament Management Staff
- [The Nomination Assessment Team](https://osu.ppy.sh/groups/7)
- [The Global Moderation Team](https://osu.ppy.sh/groups/4)
- [The Support Team](https://osu.ppy.sh/groups/22)

Reply:
I do agree we could use more clarity on this particular group of people. A distinction like World Cups staff is likely best suited. 

I think we could also discuss wheter all the staff requires to fill this criteria. Specifically map selectors and map testers. I would love to hear more feedback in that regard.